visgraph/graphcore.py

```diff
@@ -229,7 +227,7 @@ def addNode(self, nid=None, nprops=None, **kwargs):
               node and will have an ID automagically assigned.
         '''
         if nid == None:
-            nid = guid()
+            nid = uuid.uuid4().hex
```

Comment:
perf testing shows that os.urandom() (encoded or not) is over an order of magnitude faster than pythons builtin uuid module.  The paradigm here ( and in all newer synapse code ) was selected on purpose.